hw3/model.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+import numpy as np
+from sklearn.metrics import r2_score
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+cpu = torch.device("cpu")
+
+
+class RNNDenoiser(nn.Module):
```

Comment:
Seems like a really heavy architecture. Especially linear layers. Did you come up with it yourself?